### PR TITLE
Bump to PHP 8

### PR DESCRIPTION
**Pull request by ch2877**
Bump to PHP8

### CODE REVIEW

**Comment by ch2877** [comment]
@toooni 
Bumping the PHPUnit version 8 or 9 will mean that we need to bump the PHP Version to 7.2.

**Comment by toooni** [comment]
@ch2877 Yes. PHP 7.2 had it's EOL almost a year ago. And since this update does not offer new features, there is no need to keep it IMO. Don't you think?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # SwissPayment Changelog
 
+## 1.1.0 (2021-xx-yy)
+
+  * Renamed Mixed to MixedMoney (reserved word as of PHP 7).
+  * Dropped support for PHP 5.6.
```
Comment (sprain) [inline CHANGELOG.md:6]: `Dropped support for PHP 5.6.`
belongs to v1.0

1.1 would need a notice that support for PHP 7.0 to 7.1 was dropped.

**Comment by eric-reichenbach** [comment]
Fine by me, once it's merge into master we would just need to add to the change log:

"Added support for QR Bills (BankCreditTransferWithCreditorReference, BankCreditTransferWithQRR)"

As well as explicitly stating on the readme.md the compatible PHP versions

**Pull request merged**